rotkehlchen/tests/unit/test_general_cache.py

```diff
@@ -91,13 +91,20 @@ def mock_call_contract(contract, manager, method_name, **kwargs):
         target='rotkehlchen.chain.evm.contracts.EvmContract.call',
         new=mock_call_contract,
     )
+    timestamp_patch = patch(
+        target='rotkehlchen.chain.ethereum.manager.ETH_PROTOCOLS_CACHE_REFRESH',
+        new=0,
+    )
 
-    with call_contract_patch:
-        rotkehlchen_instance.update_curve_pools_cache()
+    with timestamp_patch, call_contract_patch:
```

Comment:
btw for timestamp patching no need to do this. you can use the freezegun extension we have to move time to the future